platformLibs/src/platform/ios/AudioUnit.def

```diff
@@ -0,0 +1,6 @@
+depends = AudioToolbox CFNetwork CoreAudioTypes CoreFoundation Foundation Security darwin posix
+language = Objective-C
+package = platform.AudioUnit
+modules = AudioUnit
+compilerOpts = -framework AudioUnit
+linkerOpts = -framework AudioUnit
```

Comment:
There is something wrong with AudioUnit framework. IIRC, it is effectively empty, because it just imports headers from AudioToolbox. Please check.

Reply:
Now I see that PR no longer contains this file. Might be a glitch in IDEA GitHub integration.

Reply:
No, IDEA is ok, it just gets moved to tvOS later.

Reply:
Thanks, looks like commit squashing went wrong.